sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -202,8 +214,13 @@ class GpuParquetFileFormat extends ColumnarFileFormat with Logging {
 
     val conf = ContextUtil.getConfiguration(job)
 
+    val outputTimestampType = sparkSession.sessionState.conf.parquetOutputTimestampType
     val dateTimeRebaseException = "EXCEPTION".equals(
         sparkSession.sqlContext.getConf(ShimLoader.getSparkShims.parquetRebaseWriteKey))
+    val int96RebaseException =
```

Comment:
Rather than calling this int96RebaseException, it might be simpler to evaluate it as timestampRebaseException and check for (INT96 && EXCEPTION) || (!INT96 && dateTimeRebaseException).